AppModule.xctemplate/___VARIABLE_moduleName:identifier___Module.swift

```diff
@@ -20,3 +20,74 @@ class ___VARIABLE_moduleName___Module: ___VARIABLE_moduleName___ModuleType {
         return view
     }
 }
+
+#warning("Template for the module creation spec. MOVE TO TEST TARGET")
+
+class ___VARIABLE_moduleName___ModuleSpec: QuickSpec {
+    class ___VARIABLE_moduleName___ModuleUnwrapper {
+        private let moduleView: UIViewController
+
+        var view: ___VARIABLE_moduleName___ViewController {
+            return moduleView as! ___VARIABLE_moduleName___ViewController
+        }
+
+        var presenter: ___VARIABLE_moduleName___Presenter {
+            return view.presenter as! ___VARIABLE_moduleName___Presenter
+        }
+
+        var interactor: ___VARIABLE_moduleName___Interactor {
+            return presenter.interactor as! ___VARIABLE_moduleName___Interactor
+        }
+
+        var router: ___VARIABLE_moduleName___Router {
+            return presenter.router as! ___VARIABLE_moduleName___Router
+        }
+
+        init(moduleView: UIViewController) {
+            self.moduleView = moduleView
+        }
+    }
+
+    override func spec() {
+        var view: ___VARIABLE_moduleName___ViewController!
+        var presenter: ___VARIABLE_moduleName___Presenter!
+        var interactor: ___VARIABLE_moduleName___Interactor!
+        var router: ___VARIABLE_moduleName___Router!
+
+        beforeEach {
+            let rootViewController = ___VARIABLE_moduleName___Module()
```

Comment:
```suggestion
            let rootViewController = ___VARIABLE_moduleName___Module().assemble()
```
But perhaps it would be good to extract the module itself to a `var subject: ___VARIABLE_moduleName___Module!` so it can easily be used if needed when extending the specs.

Reply:
I don't think so, there is not much you can do with the module itself, only with its content